### PR TITLE
Add Angular, Vue and React examples for the Switch

### PR DESCRIPTION
**Pull request by DokaRus**
Here, I used a code snippet extension proposed by @artem-kurchenko. Is that a priority way?

### CODE REVIEW

File: concepts/05 Widgets/Switch/10 Handle the Value Change Event.md [hunk 1]
```diff
@@ -1,16 +1,109 @@
...
-    <!--JavaScript-->
+---
+##### jQuery
+
+    <!-- tab: index.js -->
     $(function() {
-        $("#switchContainer").dxSwitch({
-            onValueChanged: function (e) {
+        $("#{widgetName}Container").dx{WidgetName}({
+             onValueChanged: function (e) {
```
Comment (AlexJameson) [inline concepts/05 Widgets/Switch/10 Handle the Value Change Event.md]:
```suggestion
            onValueChanged: function (e) {
```

File: concepts/05 Widgets/Switch/20 Keyboard Support.md [hunk 2]
```diff
@@ -30,28 +30,36 @@ You can implement a custom handler for a key using the [registerKeyHandler(key,
 ---
 ##### jQuery
 
-    <!--JavaScript-->
+    <!-- tab: index.js -->
     function registerKeyHandlers () {
-        let switch =  $("#switchContainer").dxSwitch("instance");
+        const switch =  $("#switchContainer").dxSwitch("instance");
```
Comment (AlexJameson) [inline concepts/05 Widgets/Switch/20 Keyboard Support.md:35]: Not sure if we should use const in jQuery. Better ask @RomanTsukanov about this

File: concepts/05 Widgets/Switch/00 Overview.md [hunk 3]
```diff
@@ -6,17 +6,93 @@ The **Switch** is a widget that can be in two states: "On" (when [value](/api-re
...
-     
- 
-    <!--JavaScript-->
+---
+##### jQuery
+
+    <!-- tab: index.js -->
     $(function() {
-        $("#switchContainer").dxSwitch({
+        $("#{widgetName}Container").dx{WidgetName}({
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Switch/00 Overview.md]: It is better not use {widgetName} if the file is not inherited. Please replace these constructions with the real widget name across all these documents.

File: concepts/05 Widgets/Switch/20 Keyboard Support.md (hunk 2 above)
Comment (RomanTsukanov) [reply concepts/05 Widgets/Switch/20 Keyboard Support.md:35]: According to [can i use](https://caniuse.com/#search=const), we can use `const` in this particular case.

File: concepts/05 Widgets/Switch/10 Handle the Value Change Event.md [hunk 4]
```diff
@@ -11,6 +14,96 @@ To process a new **Switch** value, you need to handle the value change event. If
         });
     });
 
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-{widget-name} (onValueChanged)="switchValueChanged($event)">
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Switch/10 Handle the Value Change Event.md]: It looks like in Angular and Vue we should show two-way binding and in React state changing.

File: concepts/05 Widgets/Switch/20 Keyboard Support.md [hunk 5]
```diff
@@ -61,13 +69,84 @@ You can implement a custom handler for a key using the [registerKeyHandler(key,
...
+        providers: [ ],
+        bootstrap: [AppComponent]
     })
+    export class AppModule { }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <Dx{WidgetName} @initialized="registerKeyHandlers" />
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Switch/20 Keyboard Support.md]: We usually call `registerKeyHandler` on `mounted`/`componentDidMount()`

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/Switch/10 Handle the Value Change Event.md (hunk 4 above)
Comment (DokaRus) [reply concepts/05 Widgets/Switch/10 Handle the Value Change Event.md]: [please notice](https://github.com/DevExpress/devextreme-documentation/pull/331#discussion_r380236985)

**Review by RomanTsukanov — APPROVED** [review]
👍

**Pull request merged**